x-pack/plugins/security_solution/public/detections/components/host_isolation/use_host_isolation_action.tsx

```diff
@@ -80,12 +80,13 @@ export const useHostIsolationAction = ({
       isIsolationAllowed &&
       isEndpointAlert &&
       isolationSupported &&
-      isHostIsolationPanelOpen === false
+      isHostIsolationPanelOpen === false &&
+      loadingHostIsolationStatus === false &&
+      agentStatus !== HostStatus.UNENROLLED
```

Comment:
need to confirm if we should still have the isolate option disabled when the host status is `UNENROLLED`